chef/cookbooks/ceph/templates/default/ceph.conf.erb

```diff
@@ -140,6 +140,9 @@
   rgw keystone admin user = <%= @keystone_settings['service_user'] %>
   rgw keystone admin tenant = <%= @keystone_settings['service_tenant'] %>
   rgw keystone admin password = <%= @keystone_settings['service_password'] %>
+<% if @keystone_settings['insecure'] -%>
+  rgw_keystone_verify_ssl = false
```

Comment:
Because our config file is using the style without underscore, maybe use `rgw keystone verify ssl`?